Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM quay.io/boundlessgeo/sonar-maven-py3-alpine
```

Comment:
if my trail of breadcrumbs is correct, this uses maven:3.5.0-jdk-8-alpine which uses openjdk:8-jdk-alpine which uses alpine3.7 which only has protobuf 3.4 available. we've been building with protobuf 3.5. i dont see a golang image that uses the alpine 3.8 (which has 3.5.2 at least)

Reply:
but maybe that doesnt matter